Cargo.toml

```diff
@@ -19,16 +19,16 @@ Event loop and I/O resources for asynchronous network services using futures
 categories = ["asynchronous", "network-programming"]
 
 [dependencies]
-crossbeam-utils = "0.6.0"
-iovec = "0.1"
-lazy_static = "1.0.2"
-log = "0.4.1"
-mio = "0.6.14"
+crossbeam-utils = "0.6.5"
+iovec = "0.1.2"
+lazy_static = "1.2.0"
+log = "0.4.6"
+mio = "0.6.16"
 mio-uds = "0.6.7"
-num_cpus = "1.8.0"
-parking_lot = "0.6.3"
-slab = "0.4.0"
-libc = "0.2.43"
+num_cpus = "1.10.0"
+parking_lot = "0.7.1"
+slab = "0.4.2"
+libc = "0.2.48"
 
 [dependencies.futures]
 version = "0.3.0-alpha.11"
```

Comment:
I forgot to update this in #66...

Reply:
Yes, how about pinning to exact `=0.3.0-alpha.13`?

Reply:
Sounds good, futures-rs is also using it (https://github.com/rust-lang-nursery/futures-rs/pull/1412).